serverless.yml

```diff
@@ -36,6 +36,7 @@ plugins:
   - serverless-s3-local
 
 custom:
+  aws-bucket-name: serverless-framework-app-janr-test-1-dev
```

Comment:
bucket name was changed to a static one due to failures with s3-local when using !Sub